### PR TITLE
Adding documentation for new features

### PR DESCRIPTION
**Pull request by krmahadevan**

### CODE REVIEW

File: testng-1.1.dtd.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<?
```
Comment (juherr) [inline testng-1.1.dtd.php:1]: No needed. Github page doesn't serve php.

It should be replaced by some js code

**Review by juherr — COMMENTED** [review]
Maybe too quick for the dtd 1.1 release if you plan some other updates soon.

**Comment by krmahadevan** [comment]
> Maybe too quick for the dtd 1.1 release if you plan some other updates soon.

I don't foresee any other addition to the DTD. Also I think we need to move away from this DTD model especially when I don't see it being helpful to end users in any way apart from helping with intellisense in their IDE.

File: testng-1.1.dtd.php (hunk 1 above)
Comment (krmahadevan) [reply testng-1.1.dtd.php:1]: So shall I get rid of this file and also the other php file as well ?

Comment (juherr) [reply testng-1.1.dtd.php:1]: Yes, you can.

**Review by juherr — APPROVED** [review]

**Pull request merged**